frontend/src/actions/submitTx.ts

```diff
@@ -27,6 +27,7 @@ export const signAndSubmitCardanoTx = async (
         txRaw: createResponse.txRaw,
         signedTxRaw,
         isFallback: createResponse.isFallback,
+        nativeTokenAmount: createResponse.nativeTokenAmount.toString(),
```

Comment:
(createResponse.nativeTokenAmount || 0).toString()

also change:
const amount = BigInt(createResponse.bridgingFee) + BigInt(values.amount);
to:
const amount = BigInt(createResponse.bridgingFee) + BigInt(createResponse.amount);